modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -109,6 +109,322 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 
 endif::is-c[]
 
+== Array Indexing 
 
+Couchbase Lite 3.2.1 introduces functionality to optimize querying arrays.
+xref:{param-module}:query-n1ql-mobile.adoc#lbl-unnest[Array `UNNEST`] to unpack arrays within a document to allow joins with the parent object, and array indexes for indexing unnested array's values to allow more efficient queries with `UNNEST`. 
 
+=== The Array Index
 
+An array index is a new type of the index for indexing nested array’s properties to allow querying with the `UNNEST` more efficiently.
+
+Below is an example array index configuration:
+
+ifdef::is-android[]
```

Comment:
This is interesting - you've essentially inlined the block-tabbed-code partial!

Reply:
Yeah I seem to have! Think it's clearer if they're all presented in the same way in this case.